src/redux/reducers/organisationUnits.js

```diff
@@ -0,0 +1,11 @@
+import { ACTION_TYPES as ACTIONS } from '../actions/organisationUnits'
+
+export default function organisationUnits(state = null, { type, payload }) {
```

Comment:
Why null? Wouldn't it be better with an empty array/obj? then there will be no need for null and empty checks. If nothing is rendered when the array is empty we don't even need any check. Also would be more type safe.